crates/viewer/re_space_view_spatial/src/proc_mesh.rs

```diff
@@ -214,6 +252,15 @@ fn generate_wireframe(key: &ProcMeshKey, render_ctx: &RenderContext) -> Wirefram
                 line_strips,
             }
         }
+        ProcMeshKey::Capsule {
+            length: _,
+            subdivisions: _,
+        } => {
+            // This is unreachable, because no visualizer asks for it yet,
+            // because it is unimplemented. Implementing it will require writing
+            // a new capsule wireframe algorithm that agrees with the solid algorithm.
+            unreachable!("wireframe capsules are not yet implemented or used")
```

Comment:
* The Rerun linter refuses to allow `unimplemented!()` — perhaps it should.
* Nitpick: It's not possible for changes in another crate to reach this code because the `proc_mesh` module is not public.
* I agree that returning an error is a better plan.

Reply:
Done.